css/styles.css

```diff
@@ -505,16 +532,148 @@ background-color: red;
 
 /* anchor */
 
-a{
-
-color: var(--call-to-action);
+a {
+  color: var(--call-to-action);
 
   text-decoration: none;
 }
 
+nav .bx-heart:hover {
+  color: var(--red);
+}
+
+@media only screen and (max-width: 768px) {
+  .homepage .k-card {
+    width: 100%;
+  }
+
+  .banner-cardsCollection-containe {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .cardsCollection {
+    display: flex;
+
+    flex-direction: column;
+
+    justify-content: center;
+  }
+
+  .banner h1 {
+    font-size: 3rem;
+
+    width: 100%;
+  }
+
+  .k-nav .k-input {
+    display: none;
+  }
+
+  .k-nav-icon a {
+    display: none;
+  }
+
+  .k-nav .k-nav-icon .bx-menu {
+    display: inline-block;
+    color: var(--call-to-action);
+  }
+
+  .summer {
+    background-image: url(/assets/bannerpic2.jpg);
+    background-size: 150%;
+    background-position: 40%;
+  }
+
+  .spring {
+    background-image: url(/assets/bannerPic4.jpg);
+
+    background-size: 150%;
+    background-position: 40%;
+  }
+
+  .sidebar {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .sidebar .k-sidebar-subsections {
+    display: none;
+  }
+
+  .ecomm-master-container {
+    display: flex;
+
+    flex-direction: column;
+    align-content: center;
+    justify-content: center;
+  }
+
+  .ecomm-master-container .ecomm-cards {
+    display: flex;
+
+    flex-direction: column;
+    width: 100%;
+  }
+
+  .ecomm-master-container .ecomm-cards .ecomm-cards-container {
+    width: 100%;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    flex-wrap: nowrap;
+  }
+
+  .ecomm-heading {
+    display: none;
+  }
 
-nav .bx-heart:hover{
+  .div-transparent {
+    transform: scale(0.7);
+  }
 
-color:var(--red);
+  .wishlist-container {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: nowrap;
+    padding: 0;
+    align-items: center;
+    justify-content: center;
```

Comment:
again, you can make separate class for it and use it directly instead of writing same code again and again